bool resetCloud
---
float64 resetCloudTime
bool success